ci/release/update-version.sh

```diff
@@ -13,14 +13,10 @@ NEXT_FULL_TAG=$1
 
 # Get current version
 CURRENT_TAG=$(git tag --merged HEAD | grep -xE '^v.*' | sort --version-sort | tail -n 1 | tr -d 'v')
-CURRENT_MAJOR=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[1]}')
-CURRENT_MINOR=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[2]}')
-CURRENT_PATCH=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[3]}')
-CURRENT_SHORT_TAG=${CURRENT_MAJOR}.${CURRENT_MINOR}
```

Comment:
These variables are used in a few rapids projects (`cudf`, `cuml`, `cuvs`) but not others.  Happy to keep them and just ignore the `shellcheck` warnings if we'd like to have these available across projects.

Reply:
There are a lot of discrepancies in the `update-version.sh` scripts across repos. We will want to keep an eye out for any best practices that we can unify across repos.

In the meantime, I am okay with removing unused variables.